lighthouse-cli/test/fixtures/seo/seo-tap-targets.html

```diff
@@ -93,13 +93,15 @@ <h1>SEO Tap targets</h1>
     <br/><br/>
 
     <!-- Only target that's being overlapped should fail, the overlapping one shouldn't -->
-    <div>
-      <a style="display: block; width: 100px; height: 30px;background: #ddd;">
-        too small target
-      </a>
-      <a style="display: block; width: 100px; height: 100px;background: #aaa;">
-        big enough target
-      </a>
+    <div style="position: relative; height: 200px;">
+      <div style="position: absolute;">
```

Comment:
maybe add a note that this is just testing our `absolute` handling ability?

I feel like we should also throw in false positive case that we handle correctly. Maybe one of the ones you were thinking about in https://github.com/GoogleChrome/lighthouse/pull/7778/files#r271846819?

Reply:
Added two more test cases in there.